### ISSUE TITLE
console prop warnings

### ISSUE THREAD

**Issue by Laviro** [ericgio/react-bootstrap-typeahead#428]
### Actual Behavior:
![selection_026](https://user-images.githubusercontent.com/26363699/53742459-50564a80-3ea1-11e9-941c-59bc890a5aef.png)

Hey guys, thanks for the good work !

- the `emptyLabel` warning got fixed.
- Do you know what could have caused the `bodyContainer` prop error?
- Is there a way to quiet the ID deprecation warning?

Thanks :hugs:

**Comment by ericgio** [comment]
These are deprecation warnings that were added in [v3.4.0](https://github.com/ericgio/react-bootstrap-typeahead/releases/tag/v3.4.0). The warnings themselves should hopefully be self-explanatory, but see the [v4.0 upgrade guide](https://github.com/ericgio/react-bootstrap-typeahead/blob/4.0/docs/Upgrading.md#v40) for more information.

**Comment by Laviro** [comment]
Thank, though even when I updated to `v3.4.0`,
and added the `id` prop.. I still see the annoying warnings in my console.:

![selection_029](https://user-images.githubusercontent.com/26363699/53763333-32a1d900-3ed3-11e9-82e2-838549ab53d0.png)

I would expect that when providing the `id`, the warning would disappear.
as for the `bodyContainer` can you add a fix for that?

Thanks

**Comment by ericgio** [comment]
Are you seeing the `bodyContainer` warning despite not setting it?

**Comment by Laviro** [comment]
yes

**Comment by ericgio** [comment]
Can you please post your code? Thanks.

**Comment by Laviro** [comment]
For example it happens with this simple mount:
```
<TypeAheadSelect
        id="test"
        options={Mocks.countries}
        labelKey="country"
        placeholder="Choose a state..."
      />
```

please check Patternfly (and thanks again for all of your work!!):  https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-3/patternfly-react/src/components/TypeAheadSelect

and TheForeman implementation for Autocomplete: https://github.com/theforeman/foreman/tree/develop/webpack/assets/javascripts/react_app/components/AutoComplete

**Comment by ericgio** [comment]
Ok I see the problem. My fault, thanks for reporting!

**Comment by Laviro** [comment]
Thanks, I will be happy to help.

**Comment by ericgio** [comment]
Fixed in v3.4.1. Thanks again for the report and sorry for the inconvenience.
